components/utils.py

```diff
@@ -562,7 +562,7 @@ def download_file(url, destination=''):
 def get_file_name_from_url(url):
     try:
         return url.split('/')[-1]
-    except:
+    except Exception:
```

Comment:
i don't think this is required at all, because i don't think `url.split('/')[-1]` can ever throw anyway

Reply:
(Also we could just use `os.path.basename` here...)